lib/package.json

```diff
@@ -7,6 +7,7 @@
   },
   "dependencies": {
     "chalk": "^4.1.0",
+    "ember-cli-addon-aware-resolver": "^1.1.0",
```

Comment:
This looks wrong; users should be able to configure whatever resolution function they want. this package should not have a dependency on a specific resolution strategy.